findIP.js

```diff
@@ -24,10 +24,11 @@ const evaluateIP = () => {
     return;
   } else {
```

Comment:
would keep variable names understandable even in small file for consistency.

Reply:
Done